app/models/email/redirect.rb

```diff
@@ -0,0 +1,12 @@
+class Email::Redirect < ApplicationRecord
+  self.table_name = 'email_redirects'
+
+  belongs_to :domain, class_name: 'Domain', foreign_key: :domain_id, inverse_of: :email_redirects
```

Comment:
@dmitry-sinina you can omit `foreign_key` key cause it will be equal to `:domain_id` by default according to Rails convention